src/common/timing.h

```diff
@@ -4,6 +4,21 @@ extern "C" {
 #endif //__cplusplus
 #include <stdint.h>
 
+/// convert ticks to nano seconds
+///
+/// @return uint64_t nanoseconds, cannot overflow
+uint64_t ticks_to_ns(uint32_t cnt);
+
+/// converts miliseconds to ticks
+///
+/// @return uint64_t ticks, cannot overflow
+uint64_t ms_to_ticks(uint32_t ms);
```

Comment:
The naming of those newly-introduced functions is misleading/not correct.

Look at the original timing.h API (without this change): 
- You have ticks and timestamps. 
- Ticks are overflowing and with undefined beginning. 
- Timestamps begin at 0 and never overflow.
- You have different functions to get ticks/timestamps with different unit: `ticks_ms()` gives you ticks with a unit of milliseconds. `ticks_us` gives you ticks with the unit of microseconds. Etc. In other words – both ticks and timestamps always have some unit (ms, us, s, ns) and the unit is given by the function you used to get the ticks/timestamps.

Given the above, what should `ticks_to_ns` do? What ticks does it convert? Ticks returned by `ticks_ns()` or ticks returned by `ticks_ms()`? 

Clearly, it is meant for some "raw ticks" of the underlying timers, that the user of the `timing` API does not even have access to.

/suggestion/ Rename the functions appropriately and make them private to the `timing` module.